### PR TITLE
Fixed string translations mentioned in the issue #2168

### PR DESCRIPTION
**Pull request by INNOVATIVEGAMER**
This commit is fix for the issue #2168 .
The strings are now getting translated with these code changes ,but currently these strings are not translated on crowdin.
So as for the issue this code actually fixes the issue on dev side only.

### CODE REVIEW

**Comment by INNOVATIVEGAMER** [comment]
> We could almost not translate `'a (multiplier: ' + value + ')'` as it's deprecated and not worth translating anymore.

So if its deprecated, shall I remove the default return statement.
I think there should be one default return statement

**Comment by Bouh** [comment]
> > We could almost not translate `'a (multiplier: ' + value + ')'` as it's deprecated and not worth translating anymore.
> 
> So if its deprecated, shall I remove the default return statement or just remove the trans tag.
> I think there should be one default return statement

You can let the line by default without trans tag indeed ;)
One day this will be delete properly with the related stuff unsused.

**Comment by INNOVATIVEGAMER** [comment]
Requested changes are applied.

File: newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js [hunk 1]
```diff
@@ -102,8 +101,8 @@ export default class ForceMultiplierField extends Component<
 export const renderInlineForceMultiplier = ({
   value,
 }: ParameterInlineRendererProps) => {
-  if (value === '1') return 'a permanent';
-  else if (value === '0' || value === '') return 'an instant';
+  if (value === '1') return <Trans>{`a permanent`}</Trans>;
+  else if (value === '0' || value === '') return <Trans>{`an instant`}</Trans>;
 
-  return 'a (multiplier: ' + value + ')';
+  return `a (multiplier: ' + ${value} + ')`;
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js:107]: This is not correct ;)

Comment (Bouh) [reply newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js:107]: The original was  

https://github.com/4ian/GDevelop/blob/bb296dc34e4af818ac45f48450b1f6691b9d0cc9/newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js#L108

Comment (INNOVATIVEGAMER) [reply newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js]: Sorry for the silly mistake. 
Template string now converted back to original string.

**Review by Bouh — APPROVED** [review]

**Comment by 4ian** [comment]
Looks good, thanks!

**Pull request merged**